Neos.Flow/Classes/Security/Policy/Role.php

```diff
@@ -75,6 +89,7 @@ public function __construct(string $identifier, array $parentRoles = [])
         $this->identifier = $identifier;
         $this->packageKey = $matches[1];
         $this->name = $matches[2];
+        $this->label = $matches[2];
```

Comment:
```suggestion
        $this->label = $matches[2];
        $this->description = '';
```
Otherwise description can be null (type mismatch to getter typehint).

Reply:
Good catch. Wouln't it be better to set the default in the property?

Reply:
Is effectively equivalent, but I kept the initializers in the constructor together in the suggestion. Whichever you prefer is fine :)

Reply:
Why not add `label` and `description` to the constructor aruguments?
I would certainly expect this and would even tend to make all setters deprecated at some point in favor of immutability

Reply:
@daniellienert @albe _ping_  :)

Reply:
Changed as suggested by @bwaidelich